sos/report/plugins/block.py

```diff
@@ -50,14 +53,19 @@ def setup(self):
             "/sys/block/loop*/loop/",
         ])
 
-        cmds = [
-            "parted -s %(dev)s unit s print",
+        udevadm_cmds = [
             "udevadm info %(dev)s",
             "udevadm info -a %(dev)s"
         ]
-        self.add_device_cmd(cmds, devices='block', blacklist='ram.*')
+
+        self.add_device_cmd(udevadm_cmds, devices='block', blacklist='ram.*',
+                            subdir=_subdir1)
+        self.add_device_cmd("parted -s %(dev)s unit s print",
+                            blacklist="ram.*", devices="block",
+                            tags="parted_s_sos", subdir=_subdir2)
```

Comment:
My first thought was "why the tag has redundant `_sos` substring, it is evident the tag belongs to `sos` - but we already had `fdisk_l_sos` tag so I am OK,